test/k8sT/Services.go

```diff
@@ -412,6 +412,67 @@ var _ = Describe("K8sServicesTest", func() {
 		})
 	})
 
+	SkipContextIf(func() bool { return !helpers.RunsOnNetNextOr419Kernel() }, "Checks local redirect policy", func() {
+		const (
+			lrpServiceName = "lrp-demo-service"
+			be1Name        = "k8s1-backend"
+			be2Name        = "k8s2-backend"
+			repeat         = 10
+		)
+
+		var (
+			deploymentYAML string
+			ctx            context.Context
+		)
+
+		BeforeAll(func() {
+			deploymentYAML = helpers.ManifestGet(kubectl.BasePath(), "lrp-test.yaml")
+			res := kubectl.ApplyDefault(deploymentYAML)
+			res.ExpectSuccess("Unable to apply %s", deploymentYAML)
+			waitPodsDs()
+			ctx, _ = context.WithTimeout(context.Background(), 1*time.Second)
```

Comment:
The `cancel` function must be kept & called to clean up context resources.

Reply:
Done.